Objects/Converters/ConverterAutocadCivil/ConverterAutocadCivilShared/ConverterAutocadCivil.Geometry.cs

```diff
@@ -871,13 +866,14 @@ public Curve SplineToSpeckle(Spline spline)
     }
 
     // handle the display polyline
-    try
+    if (spline.ToPolyline(false, true) is AcadDB.Curve poly)
```

Comment:
Can this ToPolyline() ever not be AcadDB.Curve poly? If it were, what happens downsteam of this check?

Reply:
This chunk is trying to set the `displayValue` of a spline curve, so I'll modify this to log a (noncritical) error if it fails